src/SprykerEco/Client/Payone/ClientApi/Request/AbstractRequest.php

```diff
@@ -80,11 +80,11 @@ abstract class AbstractRequest extends AbstractContainer
     protected $hash;
 
     /**
-     * @param string $encoding
+     * @param string|null $encoding
```

Comment:
Why you added null as possible value?

Reply:
It's because we have such cases, when we put something which has string|null as a return type hint to this variable.

Reply:
And we have error from PHPStan for this.